gateway/api/rest/pom.xml

```diff
@@ -33,17 +33,6 @@
       <artifactId>swagger-annotations</artifactId>
     </dependency>
     <dependency>
-      <groupId>io.swagger</groupId>
```

Comment:
@EricWittmann I do use the swagger editor in apiman, but I don't recollect I added it.. I can add it explicitly in ipaas if it's needed.